lib/recurly/pager.php

```diff
@@ -45,7 +48,7 @@ public function rewind() {
   public function current()
   {
     // Work around pre-PHP 5.5 issue that prevents `empty($this->count())`:
-    if ($this->count() == 0) {
+    if (count($this->_objects) == 0) {
```

Comment:
Seems like `if empty($this->_objects)` would be shorter. And I think then we can also ditch the comment above.